dbschema/partnership.esdl

```diff
@@ -0,0 +1,34 @@
+module default {
+  type Partnership extending Project::Child {
+    agreementStatus: Partnership::AgreementStatus;
```

Comment:
This status corresponds to the `agreement` file, as does `mouStatus` and `mou`. If you could sort those together that would be great.

The NotAttached is the default value here, so let's use that instead of null.
```suggestion
    required agreementStatus: Partnership::AgreementStatus {
    		default := Partnership::AgreementStatus.NotAttached;
    };
```